server/views/content/recent.ejs

```diff
@@ -1,21 +1,59 @@
 <div>
 
 <!--mission statement-->
-<p class="home-title">
+<!-- <p class="home-title">
```

Comment:
I think this is a typo error. Please verify

Reply:
Typo error confirmed